release_notes.md

```diff
@@ -0,0 +1,28 @@
+# Installation/Upgrade
```

Comment:
Is inclusion of this file a mistake? It has specifics about ddev v1.3.0 that I wouldn't expect to see in this PR.

Reply:
No, not a mistake. The thought process is in the PR description, but this gives us an opportunity to either build our release notes as we make changes and submit it automatically, or create a basic release notes template that we can then edit after the release is created . It looks like there are a couple outstanding references to "this upgrade" (referring to v1.3) that will definitely get removed, but the upgrade caveats are still relevant assuming we stick with MariaDB v10.2 in v1.4+.

Reply:
OK, we'll just have to make it more vanilla, as it currently contains things that would not be good for the basics of release notes.